dxp/resource_serving.md

```diff
@@ -0,0 +1,161 @@
+# Resource Serving in DXP
+
+This document explains how different resources (`.js`, `.css`, etc.) are handled by DXP to be served to the browser.
```

Comment:
Don't think you need the "." after "etc".

Reply:
:thinking: In Spanish [you do](https://www.fundeu.es/recomendacion/etc-no-va-seguido-de-puntos-suspensivos-1531/), because it's an abbreviation.

In English it looks like it depends it ends with `.` too (see https://data.grammarbook.com/blog/definitions/all-about-etc/). However the article starts with "In American English..." but then it seems to assume that it is written `etc.` for all English variants...

Reply:
Another source -> https://www.grammarly.com/blog/et-cetera-etc/

We need a [RAE](https://www.rae.es/) for English ASAP :sweat_smile:

Reply:
Why don't you use use ellipsis and forget about this… 😉

Reply:
Obviously language changes over time so it is hard to get a "definitive" answer, but over the last few decades there has been a trend in English towards eliding punctation — such as periods at the end of abbreviations — presumably to make things look "cleaner". But I imagine if you sample a number of style guides, you will find different answers (probably why the Wikipedia page says "abbreviated to etc., etc, et cet., &c., or &c").

Reply:
> Obviously language changes over time

:scream:  

> [güisqui](https://dle.rae.es/g%C3%BCisqui) :heart_eyes:

Reply:
We can stick to whatever style we decide, I don't care too much. I simply investigated it because I was curious to check if it worked differently in Spanish and English :shrug: .

Reply:
I actually learned that `etc...` is frown upon in Spanish 😂

Reply:
And in English too, really. It's kind of like saying...

https://youtu.be/djrrskjfYF0?t=13